content/blog/2020-workshop-devsprints-tickets.md

```diff
@@ -0,0 +1,37 @@
+Title: Workshop and Devsprint tickets are now open
+Date: 2020-09-15 21:00
+Author: Nikhil Maan (@sc0rpi0n101)
+Category: 2020
+Slug: 2020-workshop-devsprint-tickets
+Summary: PyCon India 2020 workshop and devsprint tickets are now open.
+
+
+We know you all have been waiting for this eagerly. Don't worry, you don't have to wait anymore! The Workshop and devsprint tickets for PyCon India are now open. 
+
+[**Go get your tickets, available now on our website**](https://in.pycon.org/2020/#ticket)
```

Comment:
We are launching the blog tonight but tickets would be available tomorrow 5PM.

I maybe wrong there. cc: @sayanchowdhury